config/environments/production.rb

```diff
@@ -1,5 +1,3 @@
-require Rails.root.join("config", "secret") if Rails.root.join("config", "secret.rb").file?
```

Comment:
Added it back. I would love to know what it is used for and why are not using `secrets.yml`.